modules/DMZ/data_kit/dataset_insight.py

```diff
@@ -30,3 +30,33 @@ def get_prediction_type(target_column):
 def get_delimiter(path):
     with open(path, 'rb') as csvfile:
         return csv.Sniffer().sniff(csvfile.read(), delimiters=';,').delimiter
+
+
+# Gets the ratio of missing values to existing values in a dataframe. Either operates on rows or columns, depending
+# on input.
+def get_missing_ratios(pandas_data, method):
+
+    if method == "column":
```

Comment:
Does !axis_toggle flip 0 to 1 and vice versa? I like that as an alternative overall